ci/prow/Makefile

```diff
@@ -33,13 +33,16 @@ config:
 
 update-config:
 	$(SET_CONTEXT)
-	$(eval TMP_YAML_CONFIG := $(shell mktemp))
-	kubectl get configmap config -o jsonpath="{.data}" 2>/dev/null >"${TMP_YAML_CONFIG}"
-	sed -i -e "s/((COMMIT_HASH_TOKEN))/$(shell git rev-parse HEAD)/" "config.yaml"
-	kubectl create configmap config --from-file=config.yaml=config.yaml --dry-run -o yaml | kubectl replace configmap config -f -
-	git checkout config.yaml
+	$(eval OLD_YAML_CONFIG := $(shell mktemp))
+	$(eval NEW_YAML_CONFIG := $(shell mktemp))
+	$(eval GCS_DEST := gs://knative-prow/configs/prow_job_config/$(shell date +%s))
```

Comment:
%s is impossible to read, better format it as "YYYY-MM-DD HH:MM:SS" or similar. You could store them inside /configs as "config_YYYY-MM-DD_HH:MM:SS.yaml" for example.

Reply:
actually I thought about that, decide not to use `YYYY-MM-DD_HH:MM:SS` as it's already a metadata for each GCS file

Reply:
Yes, it is. But it's way quicker/easier to figure that out when listing the files (through UI or CLI) instead of the extra metadata step, right? And what if I'm looking for a specific file, say from X days ago? I would have to check the metadata of a bunch of files to finally find the one I'm looking for.

Reply:
The UI is fine as the metadata is right there, while CLI is a legit use case, updated